android/src/main/java/de/simolus3/fluttie/FluttiePlugin.java

```diff
@@ -90,27 +84,22 @@ private FluttieAnimation getManagedAnimation(MethodCall call) {
 		return managedAnimations.get(id);
 	}
 
-	public RenderingThreads getRenderingThreads() {
+	RenderingThreads getRenderingThreads() {
 		return renderingThreads;
 	}
 
 	@Override
 	public void onMethodCall(MethodCall call, Result result) {
 		switch (call.method) {
 			case "isAvailable":
-				result.success(true);
+				result.success(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP);
```

Comment:
I see why we have do this, but AFAIK the cause of this crash has already been fixed in the Flutter engine and it no longer occurs on the `master` branch of flutter, right?
I'm wondering whether this is so critical that we really have to disable the plugin on pre-Lollipop... But then again we can re-enable it in a subsequent version. What do you think?

Reply:
I thought about that too :-)

I am completely unable to predict when they choose to update the beta branch. It's been five weeks now and I think this summer they went more than eight weeks without updating it. Also, as the situation is now, developers may install default Flutter, add the latest version of your fine plugin, build their app, and publish it in the store. All without a chance of knowing that they just published an app that crashes on pre-Lollipop phones. I'd rather not have that and then having to roll back this commit when they decide to release a new beta.

Reply:
That is a very good point! I'll merge this PR as it is & create a new version then, as we can still revert that part later.